### ISSUE TITLE
Handle single line comments

### ISSUE THREAD

**Issue by vjuneja** [kevinludwig/pgn-parser#4]
Single line comments start with a `;` character and go to the end of the line

https://en.wikipedia.org/wiki/Portable_Game_Notation#Comments

Sample PGN file from this url does not work.
https://zanchess.wordpress.com/2018/02/15/1st-american-chess-congress-new-york-1857/

```

;; <Paulsen // Raphael * (R3.1) 25 1-0>
;; <Raphael, Benjamin -- Paulsen, Louis  (R3.1) 25 0-1>

[Event "1st American Chess Congress"]
[Site "New York, NY USA"]
[Date "1857.10.26"]
[Round "3.1"]
[White "Raphael, Benjamin"]
[Black "Paulsen, Louis"]
[Result "0-1"]
[WhiteElo "?"]
[BlackElo "?"]
[ECO "B40"]
[EventDate "1857.10.05"]
[PlyCount "50"]

1.e4 c5 2.d4 cxd4 3.Nf3 e6 4.Nxd4 Bc5 5.Nc3 Qb6 6.Na4 Qa5+ 7.c3 Bxd4 8.Qxd4 Nf6 9.f3 Nc6 10.Qd1 O-O 11.b4 Qe5 12.Bd3 Nxb4 13.cxb4 Qxa1 14.Qc2 Qd4 15.Bb2 Qe3+ 16.Kd1 Qh6 17.Bc1 Qh4 18.Nc3 d5 19.g3 Qh5 20.Qf2 Nxe4 21.Bxe4 dxe4 22.Nxe4 Qd5+ 23.Bd2 Qxa2 24.Re1 Bd7 25.Qe3 Bb5 0-1


;; <Paulsen // Raphael   (R3.2) 47 1/2-1/2>
;; <Paulsen, Louis -- Raphael, Benjamin  (R3.2) 47 1/2-1/2>

[Event "1st American Chess Congress"]
[Site "New York, NY USA"]
[Date "1857.10.27"]
[Round "3.2"]
[White "Paulsen, Louis"]
[Black "Raphael, Benjamin"]
[Result "1/2-1/2"]
[WhiteElo "?"]
[BlackElo "?"]
[ECO "C47"]
[EventDate "1857.10.05"]
[PlyCount "93"]

1.e4 e5 2.Nc3 Nf6 3.Nf3 Nc6 4.d4 Bb4 5.d5 Ne7 6.Nxe5 d6 7.Nf3 Nxe4 8.Qd4 Bxc3+ 9.bxc3 Nf6 10.Bc4 O-O 11.O-O h6 12.Bb3 a5 13.a4 Ng6 14.Ba3 Re8 15.Rfe1 Bg4 16.Nd2 Ne5 17.c4 Re7 18.c5 Qd7 19.cxd6 cxd6 20.c4 Rae8 21.c5 Nc6 22.Rxe7 Nxd4 23.Rxd7 Nxd7 24.f3 Re2 25.Rd1 Bh3 26.c6 Rxg2+ 27.Kh1 bxc6 28.dxc6 Ne5 29.Bxd6 Nexc6 30.Bc4 Rg6 31.Bg3 Nf5 32.Ne4 Ne3 33.Rc1 Bg2+ 34.Kg1 Bxf3 35.Bd3 f5 36.Nd6 Nd4 37.Rc5 Rf6 38.Rxa5 f4 39.Nc4 fxg3 40.Nxe3 gxh2+ 41.Kxh2 g5 42.Ra7 Kf8 43.a5 Rf7 44.Ra6 Re7 45.Rf6+ Ke8 46.Nf5 Nxf5 47.Bxf5 1/2-1/2


;; <Fiske // Marache   (R1.4) 35 0-1>
;; <Fiske, Daniel -- Marache, Napoleon  (R1.4) 35 0-1>

[Event "1st American Chess Congress"]
[Site "New York, NY USA"]
[Date "1857.10.19"]
[Round "1.4"]
[White "Fiske, Daniel"]
[Black "Marache, Napoleon"]
[Result "0-1"]
[WhiteElo "?"]
[BlackElo "?"]
[ECO "A84"]
[EventDate "1857.10.05"]
[PlyCount "70"]

1.d4 e6 2.c4 f5 3.Nc3 Nf6 4.e3 b6 5.Nf3 Bb4 6.a3 Bxc3+ 7.bxc3 O-O 8.Bd3 Bb7 9.O-O Nc6 10.Bb2 Ne7 11.Nh4 d6 12.f4 Qd7 13.Qc2 Ng4 14.Rae1 Rf6 15.e4 Rh6 16.Nf3 Rf8 17.h3 Nf6 18.exf5 exf5 19.d5 c6 20.Qe2 Ng6 21.dxc6 Qxc6 22.Nd4 Qc5 23.Qe6+ Kh8 24.Bxf5 Nh5 25.Qe8 Nf6 26.Qe2 Nh4 27.Qe7 Rb8 28.Bd3 Bxg2 29.Rf2 Bxh3 30.f5 Rh5 31.Bc1 Bxf5 32.Bxf5 Nxf5 33.Nxf5 Rxf5 34.Be3 Qe5 35.Qxa7 Qg3+ 0-1


;; <Lichtenhein // Perrin   (R2.3) 30 1-0>
;; <Lichtenhein, Theodore -- Perrin, Frederick  (R2.3) 30 1-0>

[Event "1st American Chess Congress"]
[Site "New York, NY USA"]
[Date "1857.10.19"]
[Round "2.3"]
[White "Lichtenhein, Theodore"]
[Black "Perrin, Frederick"]
[Result "1-0"]
[WhiteElo "?"]
[BlackElo "?"]
[ECO "B00"]
[EventDate "1857.10.05"]
[PlyCount "59"]

1.e4 b6 2.d4 Bb7 3.Bd3 e6 4.Ne2 g6 5.O-O Bg7 6.c3 Ne7 7.f4 f5 8.e5 d6 9.Nd2 dxe5 10.fxe5 Bxe5 11.Bc4 Bd6 12.Bxe6 Bd5 13.Bxd5 Nxd5 14.Nf3 Nc6 15.Bg5 Qd7 16.Qb3 h6 17.Bh4 Na5 18.Qc2 Ne3 19.Qd3 Nxf1 20.Rxf1 g5 21.Bg3 f4 22.Qe4+ Kf7 23.Bxf4 Kg7 24.Be5+ Bxe5 25.Nxe5 Qe6 26.Rf7+ Qxf7 27.Nxf7 Kxf7 28.b4 Rae8 29.Qf3+ Kg8 30.bxa5 1-0
```

**Comment by kevinludwig** [comment]
Thanks, 
https://github.com/kevinludwig/pgn-parser/pull/5

**Comment by kevinludwig** [comment]
Now published as version 0.0.12, also; previously only comments after headers were parsed, now with this change, comments above and below header are parsed, they are stored in different keys in the object. Also comments above and below header are now stored as an array to accommodate multiple headers in each position. Ex below:
```
[
    {
        comment_above_header: [ "first comment", "second comment"],
        headers: [ ],
        comment: ["another comment, below header, prior to move text"]
    }
]
```

**Comment by vjuneja** [comment]
Thanks so much for your hard work
